platform/core/src/services/PanelService/PanelService.ts

```diff
@@ -0,0 +1,96 @@
+import { Subscription } from '../../types/IPubSub';
+import { PubSubService } from '../_shared/pubSubServiceInterface';
+
+export const EVENTS = {
+  ACTIVATE_PANEL: 'event::panelService:activatePanel',
+};
+
+export type ActivatePanelEvent = {
+  panelId: string;
+  forceActive: boolean;
+};
+
+export default class PanelService extends PubSubService {
```

Comment:
Once we are ok with this service, its usage and its API, I will add the appropriate documentation for it.